src/intervals/arithmetic.jl

```diff
@@ -218,10 +218,10 @@ function extended_div(a::Interval{T}, b::Interval{T}) where T<:Real
     S = typeof(a.lo / b.lo)
```

Comment:
I don't think this line is needed, since `a` and `b` are forced to have the same type, `S` will always be equal to the type `T`.